website/docs/cli/configuration/configuration.mdx

```diff
@@ -24,15 +24,53 @@ Think of this file as where you [bootstrap the settings or configuration of your
 [terraform](/core-concepts/components/terraform), then [this is where](/cli/configuration/components#terraform-component-behavior)
 you'd specify the command to run (e.g. [`opentofu`](/core-concepts/projects/configuration/opentofu)),
 the base path location of the components, and so forth.
+## Base Path
```

Comment:
Please remove the section lower down in the document titled `## Base Path`. This current chapter supersedes that one.

Reply:
Now, going to revisit `base_path`. Note that all the rest of the discussion here needs to be rewritten to talk about how to find the `ATMOS_CLI_CONFIG_PATH` (but is written right now for `base_path`)

---

Once we have the `ATMOS_CLI_CONFIG_PATH` determined, we will see a YAML setting of `base_path`. The `base_path` is usually relative like `./` (or just `.`)

What is that relative to when not absolute?

```
atmos --base-path . terraform plan
```

is equivalent to

```
ATMOS_BASE_PATH=. atmos terraform plan
```

T
Decision: (most precise to least)
- If `--base-path` is set (if it's relative, then it's `abs(current working directory)`
- If `ATMOS_BASE_PATH` is set, and `--base-path` is not set
- If [this is true](https://github.com/cloudposse/atmos/pull/808/files#diff-8b0c28fcf996a3ab96636e7da4845b4235522f4c1d7700896df648183c1eeae5R47), then this becomes the `base_dir` that everything is relative to.
- If we're in a git repo, then it's the root of the git repo
- If none of the above is, then it becomes abspath(./)

If the base path is set to some invalid path, atmos should fatally exit.

Reply:
- @mcalhoun opinion is that `--base-path` should supports relative paths, but `ATMOS_BASE_PATH` does not

- @osterman believes that they should be have the same way; either support it or don't (but I think we should support it)